[2022-02-15 02:28:35,166] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.model.my_new_project.stg_customers manual__2022-02-15T02:27:57.149490+00:00 [queued]>
[2022-02-15 02:28:35,199] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.model.my_new_project.stg_customers manual__2022-02-15T02:27:57.149490+00:00 [queued]>
[2022-02-15 02:28:35,200] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 02:28:35,202] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-15 02:28:35,203] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 02:28:35,224] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): model.my_new_project.stg_customers> on 2022-02-15 02:27:57.149490+00:00
[2022-02-15 02:28:35,231] {standard_task_runner.py:52} INFO - Started process 713 to run task
[2022-02-15 02:28:35,242] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dbt_training_bytecode', 'model.my_new_project.stg_customers', 'manual__2022-02-15T02:27:57.149490+00:00', '--job-id', '68', '--raw', '--subdir', 'DAGS_FOLDER/dbt-***.py', '--cfg-path', '/tmp/tmppg_8ajej', '--error-file', '/tmp/tmp0a4bgvyy']
[2022-02-15 02:28:35,244] {standard_task_runner.py:77} INFO - Job 68: Subtask model.my_new_project.stg_customers
[2022-02-15 02:28:35,340] {logging_mixin.py:109} INFO - Running <TaskInstance: dbt_training_bytecode.model.my_new_project.stg_customers manual__2022-02-15T02:27:57.149490+00:00 [running]> on host de7a240d0d46
[2022-02-15 02:28:35,428] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Bytecode
AIRFLOW_CTX_DAG_ID=dbt_training_bytecode
AIRFLOW_CTX_TASK_ID=model.my_new_project.stg_customers
AIRFLOW_CTX_EXECUTION_DATE=2022-02-15T02:27:57.149490+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-15T02:27:57.149490+00:00
[2022-02-15 02:28:35,433] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-15 02:28:35,435] {subprocess.py:74} INFO - Running command: ['bash', '-c', '\n            dbt --no-write-json run --target dev --models stg_customers             --profiles-dir /opt/***/dbt/dbt-training-bytecode-1 --project-dir /opt/***/dbt/dbt-training-bytecode-1\n            ']
[2022-02-15 02:28:35,449] {subprocess.py:85} INFO - Output:
[2022-02-15 02:28:44,612] {subprocess.py:89} INFO - 02:28:44  Running with dbt=1.0.1
[2022-02-15 02:28:45,139] {subprocess.py:89} INFO - 02:28:45  [WARNING]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2022-02-15 02:28:45,140] {subprocess.py:89} INFO - There are 2 unused configuration paths:
[2022-02-15 02:28:45,140] {subprocess.py:89} INFO - - models.jaffle_shop.marts.core
[2022-02-15 02:28:45,143] {subprocess.py:89} INFO - - models.jaffle_shop.staging
[2022-02-15 02:28:45,144] {subprocess.py:89} INFO - 
[2022-02-15 02:28:45,187] {subprocess.py:89} INFO - 02:28:45  Found 7 models, 11 tests, 0 snapshots, 0 analyses, 367 macros, 0 operations, 1 seed file, 3 sources, 0 exposures, 0 metrics
[2022-02-15 02:28:45,193] {subprocess.py:89} INFO - 02:28:45
[2022-02-15 02:28:48,708] {subprocess.py:89} INFO - 02:28:48  Concurrency: 4 threads (target='dev')
[2022-02-15 02:28:48,710] {subprocess.py:89} INFO - 02:28:48
[2022-02-15 02:28:48,727] {subprocess.py:89} INFO - 02:28:48  1 of 1 START view model dbt_bytecode_training.stg_customers..................... [RUN]
[2022-02-15 02:28:50,758] {subprocess.py:89} INFO - 02:28:50  1 of 1 OK created view model dbt_bytecode_training.stg_customers................ [SUCCESS 1 in 2.02s]
[2022-02-15 02:28:50,839] {subprocess.py:89} INFO - 02:28:50
[2022-02-15 02:28:50,841] {subprocess.py:89} INFO - 02:28:50  Finished running 1 view model in 5.64s.
[2022-02-15 02:28:50,848] {subprocess.py:89} INFO - 02:28:50
[2022-02-15 02:28:50,850] {subprocess.py:89} INFO - 02:28:50  Completed successfully
[2022-02-15 02:28:50,852] {subprocess.py:89} INFO - 02:28:50
[2022-02-15 02:28:50,854] {subprocess.py:89} INFO - 02:28:50  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2022-02-15 02:28:51,338] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-15 02:28:51,389] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=dbt_training_bytecode, task_id=model.my_new_project.stg_customers, execution_date=20220215T022757, start_date=20220215T022835, end_date=20220215T022851
[2022-02-15 02:28:51,460] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-15 02:28:51,534] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
